modules/stochastic_tools/test/tests/multiapps/commandline_control/master_wrong_multiapp_type.i

```diff
@@ -21,7 +21,7 @@
 [Samplers]
   [sample]
     type = MonteCarlo
-    num_rows = 10
+    num_rows = 2
```

Comment:
@zachmprince are you ok with this change?

Reply:
Yes I am.